tuf-spec.md

```diff
@@ -490,9 +490,11 @@ A delegated role file is located at:
 # Document formats # {#document-formats}
 
 All of the formats described below include the ability to add more
-attribute-value fields for backwards-compatible format changes.  If
-a backwards incompatible format change is needed, a new filename can
-be used.
+key-value fields to dictionary objects for backwards-compatible format
+changes.  Implementers who encounter undefined key-value pairs in the format
+must include the data when calculating hashes or verifying signatures and must
+preserve the data when re-serializing. If a backwards incompatible format change
```

Comment:
```suggestion
attribute-value fields to objects for backwards-compatible format changes.
Implementers who encounter undefined attribute-value pairs in the format
must include the data when calculating hashes or verifying signatures and must
preserve the data when re-serializing. If a backwards incompatible format change
```

The spec consistently uses attribute-value and object (rather than key-value and dictionary). Let's use the standard terms until https://github.com/theupdateframework/specification/issues/175 is addressed.